kernel/process.c

```diff
@@ -41,6 +41,10 @@ void process_init()
 	current->ktable[3] = kobject_create_graphics(&graphics_root);
 	graphics_root.count++;
 
+	printf("KOBJECT 0 INTENT: %u\n", current->ktable[0]->intent);
```

Comment:
If only for debugging, remove.

Reply:
All debug artifacts should now be removed.